README.md

```diff
@@ -111,6 +111,9 @@ const graphqlClient = MiniGraphqlHttpClient({
   // The `fetch` implementation.
   fetch: require("unfetch"), // Will attempt to use global `fetch`. You must provide `fetch` in Node.js.
 
+  // Retry
+  retry: 0, // Number of retries if HTTP request fails - network error or 5xx response
```

Comment:
Could you please follow the other code example? "What" is above, the "default" is inline.

```
  // Number of retries if HTTP request fails - network error or 5xx response
  retry: 3, // default is 0
```